File: 25 Coding Challenge I/Java/src/com/mofirojean/Challenge9.java
package com.mofirojean;

public class Challenge9 {
    // @author: mofirojean
    // @CodeChallenge 9: Create a function that will convert from Fahrenheit to Celsius

    public static void main(String[] args){
        // Executing the temptConversion method
        int n = 50;
        System.out.println(n + " F" + " is equivalent to " + temptConversion(n) + " C");
    }

    // method to convert from Fahrenheit to Celsius
    public static double temptConversion(int n){
        return (n - 32)/1.8;
    }
}

File: 50 Coding Challenge Part I/C/challenge9.c
// Coding challenge 9: Create a function that will convert from Fahrenheit to celsius

#include <stdio.h>

double fahrenhietToCelsius(int temp); // prototype function

int main(int argc, char *argv[]) {
        int temp;

        printf("Enter the temperature in celsius: ");
        scanf("%d", &temp);

        double tempCel = fahrenhietToCelsius(temp);
        printf("Temperature: %.2lf C\n", tempCel);

        return 0;
}

double fahrenhietToCelsius(int temp) {
        return (temp - 32) / 1.8;
}
